frontend/ui/bidi.lua

```diff
@@ -126,6 +127,22 @@ function Bidi.mirroredUILayout()
     return Bidi._mirrored_ui_layout
 end
 
+-- This fuction can be used by document widgets to temporarily match a
+-- widget to the document layout instead of the UI layout.
```

Comment:
to the document ~~layout~~ page turn direction instead of the UI layout direction